wesnoth-utbs.po

```diff
@@ -16109,6 +15843,10 @@ msgid ""
 "The race of this unit cannot be revealed yet. You must discover it in the "
 "game to be allowed to see its description."
 msgstr ""
+"\n"
+"\n"
+"Rasa této jednotky zatím není objevena. Musíš ji objevit ve hře, abys "
+"mohl(a) vidět její popis."
```

Comment:
Pro genderovou korektnost je podle ÚJČ vhodnější [lomítko](https://prirucka.ujc.cas.cz/?id=167), ale ještě lepší je prý (a rád souhlasím, přestože jsem to dřív bral jinak) text nekomplikovat.
```suggestion
"mohl vidět její popis."
```